test/e2e/scaffold/scaffold.go

```diff
@@ -281,7 +281,8 @@ func (s *Scaffold) APISIXGatewayServiceEndpoint() string {
 
 func (s *Scaffold) beforeEach() {
 	var err error
-	s.namespace = fmt.Sprintf("ingress-apisix-e2e-tests-%s-%d", s.opts.Name, time.Now().Nanosecond())
+	s.namespace = "ingress-apisix"
+	//s.namespace = fmt.Sprintf("ingress-apisix-e2e-tests-%s-%d", s.opts.Name, time.Now().Nanosecond())
```

Comment:
To generate the webhook certs, I have to specify the namespace of webhook service, so I make it to a constant.